_data/banking.yml

```
@@ -715,6 +715,13 @@ websites:
       facebook: nubankbrasil
       lang: pt
 
+    - name: Openbank
+      url: https://www.openbank.es
+      img: openbank.png
+      tfa: No
+      twitter: openbank_es
+      facebook: openbank.es
+
```

Since this site is in Spanish, we should add this.
```suggestion
      lang: es

```